region/maridia/inner-yellow.json

```diff
@@ -2112,6 +2112,92 @@
         {
           "from": 2,
           "to": [
+            {
+              "id": 1,
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": ["SpaceJump"]
+                },
+                {
+                  "name": "Kill one then Speedjump",
+                  "notable": false,
+                  "requires": [
+                    "SpeedBooster",
+                    "HiJump",
+                    "canWalljump",
+                    {"or":[
+                      {"and":[
+                        "Charge",
+                        {"enemyDamage": {
+                          "enemy": "Pink Space Pirate (standing)",
+                          "type": "contact",
+                          "hits": 1
+                        }}
+                      ]},
+                      {"canShineCharge": {
+                        "usedTiles": 19,
+                        "shinesparkFrames": 25,
+                        "openEnd": 1
+                      }}
+                    ]}
+                  ],
+                  "note": "If you can kill them without being hurt, you can take path 2->3->1."
+                },
+                {
+                  "name": "Kill one then IBJ",
```

Comment:
We could eliminate some duplication here with a new node representing "one killed". Maybe not mandatory right now but if not I'll at least create an issue.

Reply:
I have no other comments here so I'll just merge this and add the issue.